pyccel/codegen/printing/cwrappercode.py

```diff
@@ -280,6 +372,22 @@ def _get_wrapper_name(self, used_names, func):
 
         return wrapper_name
 
+    def get_wrapper_arguments(self, used_names):
+        """
+        Create wrapper arguments
+        Parameters
+        ----------
```

Comment:
Please add an empty line before the `Parameters` section

Reply:
Fixed : 23851c9